packages/components/src/local/organisms/planning-channel/helpers/StandardAreaMenu.tsx

```diff
@@ -49,40 +49,24 @@ const StandardAreaMenu: React.FC<StandardAreaMenuProps> = ({ areas, handler, sho
 
   /** generate the tree of activities */
   const getItems = (areas: AreaCategory[]): SelectItem[] => {
-    // TODO: this produces nested list, but I can't click them :-(
-    // const aItems = areas.map((category) => {
-    //   return {
-    //     label: category.name,
-    //     value: category.name,
-    //     items: category.areas.map((area) => {
-    //       return {
-    //         label: area.name, value: category.name + '~' + area.name
-    //       }})
-    //   }
-    // })
-    // return aItems
-
-    const flatList: SelectItem[] = []
-    areas.forEach((category) => {
-      // note: we had a crash when Serge encountered an areas block in the "old" structure.
-      // note: so, check the areas element exists
-      category.areas && category.areas.forEach((area) => {
```

Comment:
@lebaphi  - could you include this check that `category.areas` exists before running on the map please? 

There is a legacy wargame on the client network with the `previous` structure for areas.  We don't need to render it, but I don't want it to crash Serge (like it did on Tuesday morning :-o )